src/main/java/me/srgantmoomoo/postman/impl/clickgui/front/HudEditor.java

```diff
@@ -11,11 +11,12 @@
 public class HudEditor extends Module {
 	public BooleanSetting exitToClickGui = new BooleanSetting("exitToClickGui", this, true);
 	
+	private static ResourceLocation shader = new ResourceLocation("minecraft", "shaders/post/blur" + ".json");
```

Comment:
Why would u make this static?

Reply:
cause the only reference is a static one